lib/src/pages/layouts/yaru_landscape_layout.dart

```diff
@@ -47,13 +63,40 @@ class YaruLandscapeLayout extends StatefulWidget {
   State<YaruLandscapeLayout> createState() => _YaruLandscapeLayoutState();
 }
 
+const _kLeftPaneResizingRegionWidth = 4.0;
+const _kLeftPaneResizingRegionAnimationDuration = Duration(milliseconds: 250);
+
 class _YaruLandscapeLayoutState extends State<YaruLandscapeLayout> {
   late int _selectedIndex;
+  late double _leftPaneWidth;
+
+  double _initialPaneWidth = 0.0;
+  double _paneWidthMove = 0.0;
+
+  bool _isDragging = false;
+  bool _isHovering = false;
 
   @override
   void initState() {
-    _selectedIndex = widget.selectedIndex;
     super.initState();
+    _selectedIndex = widget.selectedIndex;
+    _leftPaneWidth = widget.leftPaneWidth;
+  }
+
+  @override
+  void didUpdateWidget(covariant YaruLandscapeLayout oldWidget) {
+    super.didUpdateWidget(oldWidget);
+
+    final width = MediaQuery.of(context).size.width;
+
+    // Avoid left pane to overflow when resizing the window
+    if (widget.leftPaneResizing &&
+        _leftPaneWidth >= width - widget.pageMinWidth) {
+      setState(() {
```

Comment:
`setState` could be removed because the framework is just about to call `build` after `didUpdateWidget`

Reply:
Indeed you're right.